gateway/about.toml

```diff
@@ -23,3 +23,5 @@ targets = [
     "x86_64-pc-windows-msvc",
     "x86_64-unknown-linux-musl",
 ]
+
+workarounds = ["ring"]
```

Comment:
I have no clue why this never triggered in our release pipelines. But Ring has a _weird_ license combo (nothing special in the licenses they use, but it's a long combination of OpenSSL et.al. licenses that we can't really parse without workarounds). 

https://embarkstudios.github.io/cargo-about/cli/generate/workarounds.html#ring